apps/source-storybook/src/README.md

```diff
@@ -0,0 +1,3 @@
+# Docs
+
+Please see our [Storybook docs](https://guardian.github.io/source)
```

Comment:
Is there a reason to link out of here rather than having the docs in this readme?

Reply:
Good point. That links to the Storybook that is currently on the old Source repo. We can look into what to replace it with when the time comes.

Reply:
I've deleted this file as it sits more in docs than in the Storybook app.